app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  has_one :user, through: :post
```

Comment:
- [ ] Comments should simply belong to user same as post.

Reply:
We have added the user to comments